dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/FirstNamesTest.java

```diff
@@ -0,0 +1,59 @@
+package edu.harvard.iq.dataverse.export.openaire;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author francesco.cadili@4science.it
+ */
+public class FirstNamesTest {
```

Comment:
We should keep tests separate from non tests code:
`dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/openaire/FirstNamesTest.java` -> `dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/FirstNamesTest.java`